llvm/test/MC/Disassembler/AMDGPU/gfx11_dasm_vop2_dpp16.txt

```diff
@@ -1,1750 +1,1949 @@
-# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -disassemble -show-encoding < %s | FileCheck -check-prefixes=GFX11,W32 %s
-# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -mattr=+wavefrontsize64 -disassemble -show-encoding < %s | FileCheck -check-prefixes=GFX11,W64 %s
+; NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py UTC_ARGS: --version 5
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -disassemble -show-encoding < %s | FileCheck -check-prefixes=GFX11,W32,GFX11-REAL16 %s
```

Comment:
The prefixes come in a surprising order. Why is it not `GFX11,GFX11-REAL16,W32`?

Reply:
I think both will work? If there are common checklines they will be grouped to the same tag

Reply:
The order is significant. You should have the most matching cases first

Reply:
I think for the W32/W64 vs TRUE16/FAKE16 it might be hard to tell which one is the most matching case. The number might be close.

Also, the script that update this test run a sorting on the prefixes and use the one with the largest number of matches so I think it should be fine

Reply:
Was that a recent change? I know I've struggled with tests where the order was not correct

Reply:
Hi Matt for the script it's not a recent change as it should be there from the beginning. If you saw a problem then there might be a bug. Can you share the case to me when you see it next time? Thanks!

For this patch, are you ok that we just merge with this order?

Reply:
Oh, this isn't update_llc_test_checks